src/component/actions/ChangeGroups/RemoveGroupsForm.tsx

```diff
@@ -102,7 +102,7 @@ export class RemoveGroupsForm extends React.PureComponent<
             groupEl = (
                 <GroupsElement
                     ref={this.props.onBindWidget}
-                    name="Group"
+                    name="Groups"
```

Comment:
think this makes more sense as `GroupElement` became `GroupsElement`. might make sense to do the same for `FieldElement`, `FlowsElement`?